activemq/CVE-2015-5254/README.md

```diff
@@ -40,16 +40,22 @@ java -jar jmet-0.1.0-all.jar -Q event -I ActiveMQ -s -Y "touch /tmp/success" -Yp
 
 ![](1.png)
 
-At this point, a queue named event will be added to the target ActiveMQ. We can visit `http://your-ip:8161/admin/browse.jsp?JMSDestination=Event` to see all messages in this queue:
+At this point, a queue named event will be added to the target ActiveMQ. 
+
+You can visit `http://your-ip:8161/admin/browse.jsp?JMSDestination=Event` to see all messages in this queue 
+
+( __login / password:__  admin / admin ):
```

Comment:
Two asterisks are better than underscores here, underscore are not always considered to be "bold" in many Markdown parsers.

Reply:
done